File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        c=0
        d={}
        for i in s:
            if i in d:
                d[i]+=1
            else:
                d[i]=1
        for i in t:
            if i in d:
                d[i]-=1
            else:
                d[i]=-1
        for i in d.values():
            if(i<0):
                c+=(i*(-1))
        return c

File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        map<int,int> mp;
        int k=0,ans=0;
        for(int i=0;i<s.size();i++)
        {
            mp[s[i] - 'a']++;
            mp[t[i] - 'a']--;
        }
        // for(int i=0;i<26;i++)
        // {
        //     ans+=mp[i];
        // }
        for ( auto it : mp){ 
            if (it.second < 0) ans += (it.second * -1);
        }

        return ans;
    
    }
};